frontend/class-frontend.php

```diff
@@ -1157,11 +1157,28 @@ public function publisher() {
 	}
 
 	/**
+	 * Sanitizes keywords and returns either an empty string or sanitized
+	 * keywords.
+	 *
+	 * @param type $keywords
```

Comment:
Could you please add a description for the `$keywords` parameter?
Our coding standards require a description to be present here.